### PR TITLE
Add getInvoicesCollection

### PR DESCRIPTION
**Pull request by luigifab**
### Description

This PR is useless, it add `$order->getInvoicesCollection()` as an alias for `getInvoiceCollection()`.
Like `getShipmentsCollection()`, `getCreditmemosCollection()` and `getTracksCollection()`.

OpenMage 20.0.16 / PHP 8.0.25

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by elidrissidev — APPROVED** [review]

**Comment by fballiano** [comment]
I've checked in the codebase and all of the "getSomethingCollection" method has a singular naming, so I wouldn't merge this specific PR since it would be the exception, sorry.

**Comment by elidrissidev** [comment]
> I've checked in the codebase and all of the "getSomethingCollection" method has a singular naming

I see the opposite:
https://github.com/OpenMage/magento-lts/blob/fb8372940f9d27eefa10ff41be85d438e74aee42/app/code/core/Mage/Sales/Model/Order.php#L2117

https://github.com/OpenMage/magento-lts/blob/fb8372940f9d27eefa10ff41be85d438e74aee42/app/code/core/Mage/Sales/Model/Order.php#L2137

https://github.com/OpenMage/magento-lts/blob/fb8372940f9d27eefa10ff41be85d438e74aee42/app/code/core/Mage/Sales/Model/Order.php#L2156

https://github.com/OpenMage/magento-lts/blob/fb8372940f9d27eefa10ff41be85d438e74aee42/app/code/core/Mage/Sales/Model/Order.php#L2175

**Comment by fballiano** [comment]
Schermata 2022-11-11 alle 09 17 13

Schermata 2022-11-11 alle 09 17 24

I guess it's random then :-D

File: app/code/core/Mage/Sales/Model/Order.php [hunk 1]
```diff
@@ -2129,6 +2129,11 @@ public function getInvoiceCollection()
         return $this->_invoices;
     }
 
+    public function getInvoicesCollection()
```
Comment (elidrissidev) [inline app/code/core/Mage/Sales/Model/Order.php:2132]: I'd suggest adding a doc block here otherwise I'm sure @sreichel will do it later :D

Comment (sreichel) [reply app/code/core/Mage/Sales/Model/Order.php:2132]:
```suggestion
    /**
     * Retrieve order invoices collection
     *
     * @return Mage_Sales_Model_Resource_Order_Invoice_Collection
     */
    public function getInvoicesCollection()
```

**Review by elidrissidev — APPROVED** [review]

**Review by sreichel — APPROVED** [review]

**Comment by sreichel** [comment]
Fixed conflict.

**Review by elidrissidev — APPROVED** [review]

**Review by sreichel — APPROVED** [review]

**Pull request merged**